prerequisites.sh

```diff
@@ -0,0 +1,20 @@
+set -e
+
+if [ -d src/libsemigroups ]; then
+  echo -e "a version of libsemigroups is already present; no action required"
```

Comment:
Wouldn't it be safer to check if the correct version of Libsemigroups is installed? This is what happens in the configuration step for Semigroups. This is done by `tr -d '\n' < src/libsemigroups/.VERSION` (after checking that the `.VERSION` file exists.